ts/Core/Globals.ts

```diff
@@ -75,6 +75,11 @@ declare global {
      * @todo: Rename UMD argument `win` to `window`
      */
     const win: Window|undefined;
+
+    interface Window {
+        moment: any;
+        Date: any;
+    }
```

Comment:
No more `any` types in Globals (only `AnyRecord`). Delete this interface definition and cast the `win`/`window` object to `AnyRecord`, if the code access the `moment` or `Date` properties in an unsafe way.